internal/metrics/metrics.go

```diff
@@ -22,7 +22,7 @@ type metrics struct {
 	kmmoResourcesNum prometheus.Gauge
 }
 
-func New() Metrics {
+func New() *metrics {
```

Comment:
Sorry. Another small one. Do we want to have a more meaningful name? Let say `NewMetrics`?

Reply:
I am not sure about it.  I think that package name allows to uniquely identify the function. The problem arises when we have more then one interface/struct in the same package